### ISSUE TITLE
Focus order is not in logical manner after "Close" button on Graph-Explorer page.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#750]
**Describe the bug**
Users who rely on keyboards will affect to navigate, if focus order is not in logical manner after  "Close" button on Graph-Explorer page.

**To Reproduce**
Step 1: Open the Graph Explorer URL in Edge browser.
Step 2: Navigate to "Close" button on Graph-Explorer page..
Step 3: Now press Tab key and Observe whether focus indicator is in logical manner.

Focus order is not in logical manner after "Close" button .
After "Close" button, focus is moving to “Share query” button instead of moving “Response preview” button.

**Expected behavior**
Focus order should be in logical manner after "Close" button .
After "Close" button, focus should move to “Response preview” button instead of moving  “Share query” button.

**Desktop (please complete the following information):**
OS: OS build Version 2004 (OS Build 20170.1000)
URL: Graph Explorer - Microsoft Graph
Browser: Anaheim Edge Dev Version: 87.0.634.0 (Official build) dev (64-bit)
